src/toil/test/src/jobCacheTest.py

```diff
@@ -350,7 +350,8 @@ def _parseAssertionError(logFile):
                 for line in logFileHandle:
                     line = line.strip()
                     if workerLogName is None:
-                        if line.startswith('Reporting'):
+                        if line.startswith('The jobWrapper seems to have left '
```

Comment:
Yeah, unfortunately this is critical to the testCacheEvictionFailCase. It will fail if it can't find this line or, subsequently, the assertion in the log output caused by the caching failure.